package.json

```diff
@@ -68,8 +68,10 @@
     "@commitlint/cli": "^11.0.0",
     "@commitlint/config-conventional": "^11.0.0",
     "@contentful/browserslist-config": "^1.0.1",
-    "@parcel/babel-plugin-transform-runtime": "2.0.0-nightly.1823",
-    "@parcel/babel-preset-env": "2.0.0-nightly.526",
+    "@parcel/babel-plugin-transform-runtime": "2.0.0-nightly.2233",
+    "@parcel/babel-preset-env": "2.0.0-nightly.612",
+    "@parcel/transformer-typescript-types": "2.0.0-nightly.612",
```

Comment:
👀 `nightly`?

Reply:
yeah, official parcel@2 is not ready yet, but it's used by `react-spectrum` as nightly and in general works well :) As soon they are out with official parcel@2 we make a switch too.